### ISSUE TITLE
Bug when multiple IDs are specified in the `for` attribute of the `<output>` element

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#1611]
- Markuplint version: `4.6.1`
- Node.js version: `20.11.0`
- OS: Windows

## Describe the bug

The `for` attribute of the `<output>` element is <q>a string consisting of an unordered set of unique space-separated tokens</q>. 👉[HTML spec](https://html.spec.whatwg.org/multipage/form-elements.html#the-output-element)

## Code Example

```html
<p><label>1 <input id="1" /></label> + <label>2 <input id="2" /></label></p>
<p><output for="1 2"></output></p>
```

## The current behavior

```
<markuplint> error: 期待されないホワイトスペースが発見されました。属性「for」はID形式である必要があります (https://html.spec.whatwg.org/multipage/dom.html#global-attributes:concept-id) (invalid-attr)
<markuplint> error: ID「1 2」がありません (no-refer-to-non-existent-id)
```

## The expected behavior

No Error
